src/routes/patterns/components/ImageLists.js

```diff
@@ -1,13 +1,13 @@
 import React, { Component } from "react";
-import { ImageList } from "components/baseline";
+import { Button, ImageList } from "components/baseline";
 
 const references = [
-  { url: require("./content/references/01.jpg"), name: "1" },
-  { url: require("./content/references/02.png"), name: "2" },
-  { url: require("./content/references/03.jpg"), name: "3" },
-  { url: require("./content/references/05.jpg"), name: "5" },
-  { url: require("./content/references/06.jpg"), name: "6" },
-  { url: require("./content/references/04.jpg"), name: "4" }
+  { url: require("./content/references/01.jpg"), name: "1", size: "long" },
```

Comment:
@shiftsave this property conflicts with one we already have when we get the images from Dropbox. They have the size property already which is bytes. Maybe we should name this `aspectRatio`?